workload/src/main/java/com/google/finapp/WorkloadMain.java

```diff
@@ -0,0 +1,154 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.finapp;
+
+import com.google.protobuf.ByteString;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.StatusRuntimeException;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
+/**
+ * A workload generator for the finance sample app that creates traffic by sending gRPC requests to
+ * the server.
+ */
+public final class WorkloadMain {
+  private static final String DEFAULT_ACCOUNT_BALANCE = "10000";
+  private static final String DEFAULT_TRANSFER_AMOUNT = "20";
```

Comment:
not a need, but in addition to a default transfer amount, we could have some equation that takes a percentage or amount from the balance. like having the transfer amount be equal to `account_balance - (account_balance * 0.01)` to make the transfer amounts different every time or something similar.